### ISSUE TITLE
HELP！I encountered some problems while training the model

### ISSUE THREAD

**Issue by XieGang729** [tensorflow/models#9849]
URL：
https://github.com/tensorflow/models/tree/master/research/delf/delf/python/training

@andrefaraujo ，Hi！How's everything going?When training the DELG model, I encountered two problems, I hope you can help me solve it，Thank you！

1.The data set you use for training is GLD v2. According to the improvement you give, 500 training sets should be used, right?Can the training set be smaller? How small can it be?

2.If I want to use my own data set, how should I label the data set? In the GLD v2 data set file, where can I see the label file?

THANK YOU! XD

**Comment by XieGang729** [comment]
@andrefaraujo ,would you like to help me? thanks a lot!

**Comment by andrefaraujo** [comment]
1) Sorry, not sure what you mean by "500 training sets". You can see details about the GLDv2 training set at: https://github.com/cvdfoundation/google-landmark

2) In [this section of the README](https://github.com/tensorflow/models/tree/master/research/delf/delf/python/training#prepare-the-data-for-training), you will see the command that creates training data in TFRecords format. The TFRecords will contain information about image-level labels (specifically, [see this description](https://github.com/tensorflow/models/blob/master/research/delf/delf/python/training/build_image_dataset.py#L43)).

Thanks!
